common/src/WooCommerce.php

```diff
@@ -143,6 +133,29 @@ public function order_completed($order_id) {
         }
     }
 
+    public function gtin_wwk_option() {
```

Comment:
`wwk` does not belong in the name of this method or anywhere else in the common plugins source code.

Reply:
Also, you can write new methods using the usual `camelCaseNaming`, this code is just really old and I didn't change that when doing the TrustProfile port.